### ISSUE TITLE
REPL mode error on backspace

### ISSUE THREAD

**Issue by Keno** [JuliaLang/Pkg3.jl#6]
```
pkg> up JSOERROR (in the keymap): type Void has no field options
Stacktrace:
 [1] edit_backspace(::Base.LineEdit.PromptState) at .\repl\LineEdit.jl:609
 [2] edit_backspace(::Base.LineEdit.MIState) at .\repl\LineEdit.jl:178
 [3] (::getfield(Base.REPL, Symbol("##18#21")){Base.LineEdit.Prompt})(::Base.LineEdit.MIState, ::Void, ::Vararg{Any,N} where N) at .\repl\REPL.jl:701
 [4] (::getfield(Base.LineEdit, Symbol("##19#20")){getfield(Base.REPL, Symbol("##18#21")){Base.LineEdit.Prompt},String})(::Base.LineEdit.MIState, ::Void) at .\repl\LineEdit.jl:1059
 [5] prompt!(::Base.Terminals.TTYTerminal, ::Base.LineEdit.ModalInterface, ::Base.LineEdit.MIState) at .\repl\LineEdit.jl:2061
 [6] run_interface(::Base.Terminals.TTYTerminal, ::Base.LineEdit.ModalInterface) at .\repl\LineEdit.jl:1962
 [7] run_frontend(::Base.REPL.LineEditREPL, ::Base.REPL.REPLBackendRef) at .\repl\REPL.jl:986
 [8] run_repl(::Base.REPL.LineEditREPL, ::getfield(Base, Symbol("##542#543"))) at .\repl\REPL.jl:182
 [9] _start() at .\client.jl:439
```
This is on 0.7

**Comment by StefanKarpinski** [comment]
Oh, I haven't tested on 0.7 at all – I shocked it even loads 😂

**Comment by KristofferC** [comment]
Fixed by #30
